module/os/windows/spl/spl-taskq.c

```diff
@@ -2440,7 +2440,7 @@ taskq_create_common(const char *name, int instance, int nthreads, pri_t pri,
 	 * Make sure the name is 0-terminated, and conforms to the rules for
 	 * C indentifiers
 	 */
-	(void) strncpy(tq->tq_name, name, TASKQ_NAMELEN + 1);
+	(void) strlcpy(tq->tq_name, name, TASKQ_NAMELEN + 1);
```

Comment:
since the next call passes TASKQ_NAMELEN + 1 I'm assuming that's the size of the buffer

Reply:
Linux went with (void) strlcpy(tq->tq_name, name, sizeof (tq->tq_name)); 
which is actually a good idea, change header and code dont need changing

Reply:
don't forget the related changes in the header files?

feel free to delete the /* ARGSUSED*/ comments